twoliter/embedded/Makefile.toml

```diff
@@ -137,6 +137,8 @@ TESTSYS_LOG_LEVEL = "info"
 # Certain variables are defined here to allow us to override a component value
 # on the command line.
 
+BUILDSYS_VERSION_IMAGE = { script = ["awk -F '[ =\"]+' '$1 == \"version\" {print $2}' ${BUILDSYS_RELEASE_CONFIG_PATH}"], condition = { env_not_set = ["BUILDSYS_VERSION_IMAGE"]}}
```

Comment:
I think we could require this to be set instead of discovering it from Release.toml here. This code is duplicated in Bottlerocket's Makefile.toml which means that Bottlerocket's build is always going to be setting this.

Reply:
> I think we could require this to be set instead of discovering it from Release.toml here. This code is duplicated in Bottlerocket's Makefile.toml which means that Bottlerocket's build is always going to be setting this.

Note, we discussed, in `twoliter make` we can look for `BUILDSYS_VERSION_IMAGE`, if it's not there or empty, we can get it from Twoliter.toml and set it before calling `cargo make`.